src/FunctionApps/python/FhirServerExport/__init__.py

```diff
@@ -19,11 +19,9 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
 
     cred_manager = fhir.AzureFhirserverCredentialManager(fhir_url=fhir_url)
```

Comment:
Flagging this as something that will need to be changed once the refactor lands. It'll become `azure.AzureFhir...`.